lib/node_modules/@stdlib/blas/base/cscal/README.md

```diff
@@ -0,0 +1,243 @@
+
+var im = imagf( z );
+// returns 14.0
+```
+
+#### cscal.ndarray( N, cx, x, strideX, offsetX )
+
+Scales values from `x` by `cx` using alternative indexing semantics.
+
+```javascript
+var Complex64Array = require( '@stdlib/array/complex64' );
+var realf = require( '@stdlib/complex/realf' );
+var imagf = require( '@stdlib/complex/imagf' );
+
+var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+var cx = new Complex64Array( [ 2.0, 2.0 ] );
+
+cscal.ndarray( x.length, cx, x, 1, 0 );
+
+var z = x.get( 0 );
+// returns <Complex64>
+
+var re = realf( z );
+// returns -2.0
+
+var im = imagf( z );
+// returns 6.0
+```
+
+The function has the following additional parameters:
+
+-   **offsetX**: starting index for `x`.
+
+While [`typed array`][mdn-typed-array] views mandate a view offset based on the underlying buffer, the offset parameters support indexing semantics based on starting indices. For example, to scale every other value in `x` starting from the second value by `cx` where `x[i] = cx[0] * x[i]`, `x[i+1] = cx[1] * x[i+1]`,...,
+
+```javascript
+var Complex64Array = require( '@stdlib/array/complex64' );
+var realf = require( '@stdlib/complex/realf' );
+var imagf = require( '@stdlib/complex/imagf' );
+
+var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0 ] );
+var cx = new Complex64Array( [ 2.0, 2.0 ] );
+
+cscal.ndarray( 2, cx, x, 2, 1 );
+
+var z = x.get( 3 );
+// returns <Complex64>
+
+var re = realf( z );
+// returns -2.0
+
+var im = imagf( z );
+// returns 30.0
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   If `N <= 0` or `strideX <= 0` , both functions return `x` unchanged.
+-   `cscal()` corresponds to the [BLAS][blas] level 1 function [`cscal`][cscal].
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var Complex64Array = require( '@stdlib/array/complex64' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
+var Complex64 = require( '@stdlib/complex/float32' );
+var cscal = require( '@stdlib/blas/base/cscal' );
+
+function rand() {
+    return new Complex64( discreteUniform( 0, 10 ), discreteUniform( -5, 5 ) );
+}
+
+var x = filledarrayBy( 10, 'complex64', rand );
+console.log( x.get( 0 ).toString() );
+var cx = new Complex64Array( [ 2.0, 2.0 ] );
+
+// Scales elements from `x` by `cx`:
+cscal( x.length, cx, x, 1 );
+console.log( x.get( x.length-1 ).toString() );
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/blas/base/cswap`][@stdlib/blas/base/cswap]</span><span class="delimiter">: </span><span class="description">interchanges two complex single-precision floating-point vectors.</span>
```

Comment:
```suggestion
```

Don't manually populate this section. For any new package, contents should be removed.